integration/inspektor-gadget/testdata/prometheus/counter.yaml

```diff
@@ -0,0 +1,11 @@
+metrics:
```

Comment:
This is missing `metrics_name`. Also, we need to update the documentation to include `metrics_name` everywhere?

Reply:
oh right, I only updated the part of the guide. Thanks!